apps/user/src/app/global.css

```
@@ -77,5 +115,1506 @@
+}
+
+/* Header */
+.nf-card-header {
+  display: flex;
+  align-items: flex-start;
+  justify-content: space-between;
+  gap: 12px;
+}
+
+.nf-word {
+  font-family: 'Geist', sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Geist" (font-family-name-quotes)

(font-family-name-quotes)